src/float/conv.rs

```diff
@@ -268,6 +233,87 @@ intrinsics! {
     pub extern "C" fn __fixunsdfdi(f: f64) -> u64 {
         float_to_int(f)
     }
+}
+
+// The ABI for the following intrinsics changed in LLVM 14. On Win64, they now
+// use Win64 ABI rather than unadjusted ABI. Pick the correct ABI based on the
+// llvm14-builtins-abi target feature.
+
+#[cfg(target_feature = "llvm14-builtins-abi")]
```

Comment:
Should this target feature be behind a feature gate?

Reply:
Apparently, we don't feature-gate `#[cfg(target_feature)]` (though we do gate `#[target_feature]`). `cfg(target_feature)` exposes all unstable target features on nightly builds.